### DISCUSSION TITLE
How to introduce gosl on MACOS?

### DISCUSSION THREAD

**Discussion by broodning** [cpmech/gosl#59] [Q&A]
I have installed all the required packages like gcc. But after running command "./all.bash", the program failed and returned "./oblas.go:23:10: fatal error: 'lapacke.h' file not found".
It is possible to use gosl in MACOS ?

**Comment by cpmech** [comment]
Hi, the use in macOS is quite tricky... Your problem in this case is the missing `liblapacke-dev` (that comes with Debian/Ubuntu).
